src/lib/metadataAPI.js

```diff
@@ -14,17 +17,25 @@ const metadataAPI = {
             return Promise.reject(new Error('id and template are required parameters'));
         }
 
-        return metadataAPI.getMetadata(id, 'global', template, options).catch((err) => {
-            const { response } = err;
-            // If the http response is 404, this is a valid case because the metadata template
-            // may not be initialized on the requested file. Resolve the promise with
-            // a constructed hasxrefs value
-            if (response && response.status === 404) {
-                return Promise.resolve({ hasxrefs: 'false' });
-            }
-
-            throw err;
-        });
+        return metadataAPI
+            .getMetadata(id, SCOPE_GLOBAL, template, options)
+            .then((response) => {
+                // The hasxrefs value is returned as a string 'false' or 'true' so we want
+                // to convert this to a boolean
+                const { [FIELD_HASXREFS]: hasxrefsValue } = response;
+                return { ...response, [FIELD_HASXREFS]: hasxrefsValue === 'true' };
+            })
+            .catch((err) => {
```

Comment:
do we need this? We we're not doing anything with our error anyways.

Reply:
🤔 yeah you're right, even if it failed with 400 or any other error code we would just not show a banner. I'll remove the code and related unit tests